bin/pacbio_barcode_check.py

```diff
@@ -45,19 +47,28 @@ def detect_barcodes_from_read_file_names(barcodes_fasta_path, pacbio_read_files)
 
 def check_if_barcodes_exist_in_barcodes_fasta(barcodes_list, barcodes_fasta_path):
     """
-    Checks if the specified barcodes exist in the barcode sequences FASTA file, exits with an error message if a barcode is not found
+    Checks if the specified barcodes exist in the barcode sequences FASTA file, prints a message for each missing barcode.
```

Comment:
If this change is made, the script's behaviour no longer matches Damon's script description in the header (`If any User-supplied barcodes arn't in data then pipeline dies`)

Reply:
Reverted to previous version of script